bindings/nodejs/lib/index.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+export * from './client';
+export * from './wallet';
+export * from './secretManager';
+export * from './utils';
+export * from '../types/secretManager';
```

Comment:
What's that? Can't it be with `secretManager` ?

Reply:
Moved it